src/apns.erl

```diff
@@ -207,6 +210,8 @@ default_headers([], Headers) ->
   Headers;
 default_headers([Key | Keys], Headers) ->
   case application:get_env(apns, Key) of
+    undefined ->
+      default_headers(Keys, Headers);
```

Comment:
I've added this clause but I'm not sure it's necessary, it just seems weird that it's not there?

Reply:
Maybe we can do this…
```erlang
  case application:get_env(apns, Key, undefined) of
    undefined ->
      default_headers(Keys, Headers);
  Value ->
      NewHeaders = Headers#{Key => to_binary(Value)},
      default_headers(Keys, NewHeaders)
  end.
```

Reply:
Yeah, done.